jenkins-job-generator/definitions/build-jobs.yml

```diff
@@ -531,13 +531,35 @@
                         build job: 'generate-cf-java-buildpack-data', propagate: false, wait: false
                     }}
                 }}
+                stage('Build Debian Linux Packages') {{
+                    when {{
+                        expression {{ env.VERIFICATION_RESULT != "2" && params.PUBLISH == true && params.RELEASE == true }}
+                    }}
+                    stage("Checkout Infrastructure Repository") {
```

Comment:
Checkout Infrastructure Repository is not necessary here. It's already available in the build job.